<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeResultWindow</class>
 <widget class="QWidget" name="NodeResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="nodeResult"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sorted BY：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showForLabel">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Platform：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="platformSortBtns"/>
       </item>
       <item>
        <widget class="QRadioButton" name="nameRadBtn">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="playsRadBtn">
         <property name="text">
          <string>plays</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="commentsRadBtn">
         <property name="text">
          <string>comments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="durationRadBtn">
         <property name="text">
          <string>duration</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
